src/m3ninx/README.md

```diff
@@ -0,0 +1,11 @@
+# m3ninx [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov]
+Reverse index.
+
+<hr>
+
+This project is released under the [Apache License, Version 2.0](LICENSE).
```

Comment:
Might need to fix the reference to the license file here? Not sure.

Reply:
will update the README appropriately

Reply:
done